index.html

```diff
@@ -27,6 +27,5 @@ <h2>All dogs:</h2>
     </section>
```

Comment:
This may be relevant for line 9:

We can break up long html lines by putting attributes on different lines like this:

```html
<input
    name="dogBreed"
    id="searchByDogbreed"
    type="text"
    placeholder="Enter dogbreed here"
>
```